streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java

```diff
@@ -113,6 +113,6 @@ public RestResponse handleConstraintViolationException(ConstraintViolationExcept
   @ExceptionHandler(value = UnauthorizedException.class)
   @ResponseStatus(HttpStatus.FORBIDDEN)
   public void handleUnauthorizedException(Exception e) {
-    log.info("Permission denied，{}", e.getMessage());
+    log.error("Permission denied，{}", e.getMessage());
```

Comment:
If e is NullPointerException, then e.getMessage will throw nullPointExp...  You can test this:

```
  @Test
  public void testException() {
    try {
      throw new NullPointerException();
    } catch (Exception e) {
      // e.getMessage will be throw nullExp....
      System.out.printf(e.getMessage());
    }
  }
```

Reply:
done